packages/app-page-builder/src/editor/components/Element.tsx

```diff
@@ -100,6 +100,19 @@ const Element = (props: ElementProps) => {
         return null;
     }
 
+    let renderedPlugin = null
```

Comment:
I'd still ask you to extract this into a separate function  at the top of the file, the way `getElementPlugin` on line `39` is done.